prefect_kubernetes/utilities.py

```diff
@@ -35,6 +37,37 @@ def add(self, value):
         super().add(value)
 
 
+def enable_socket_keep_alive(client: ApiClient) -> None:
+    """
+    Setting the keep-alive flags on the kubernetes client object.
+    Unfortunately neither the kubernetes library nor the urllib3 library which
+    kubernetes is using internally offer the functionality to enable keep-alive
+    messages. Thus the flags are added to be used on the underlying sockets.
+    """
+    import socket
```

Comment:
Tiniest of nits - I think we should move this to the top of the file unless it's causing a circular import.

Reply:
I blame the original author 😜